addons/account/tests/test_chart_template.py

```diff
@@ -363,3 +382,27 @@ def local_get_data(self, template_code):
         self.assertFalse(parent_tax.active, "The parent tax should be inactive.")
         self.assertEqual(len(children_taxes), 2, "Two children should have been created, even if they are inactive.")
         self.assertEqual(children_taxes.mapped('active'), [False] * 2, "Children taxes should be inactive.")
+
+    def test_update_reload_no_new_data(self):
+        """ Tests that the reload does nothing when data are left unchanged.
+        Tested models: account.group, account.account, account.tax.group, account.tax, account.journal,
+        account.reconcile.model, account.fiscal.position, account.fiscal.position.tax, account.tax.repartition.line,
+        account.account.tag.
+        """
+        def get_domain(model):
+            if model == 'account.account.tag':
+                return 'country_id', '=', self.company_1.country_id.id
```

Comment:
This is returning one leaf, not a domain like the function name suggests
```suggestion
                return [('country_id', '=', self.company_1.country_id.id)]
```